botstrap.py

```diff
@@ -126,6 +179,24 @@ def create_webhook(name: str, channel_id_: int, client: DiscordClient) -> str:
 
     config_str += "\n#Channels\n"
 
+    rules_channel_id = all_channels[RULES_CHANNEL_NAME]
+    announcements_channel_id = all_channels[ANNOUNCEMENTS_CHANNEL_NAME]
+
+    upgrade_server_to_community_if_necessary(GUILD_ID, rules_channel_id, announcements_channel_id, discord_client)
+
+    create_help_channel = True
+
+    if PYTHON_HELP_CHANNEL_NAME in all_channels:
+        python_help_channel_id = all_channels[PYTHON_HELP_CHANNEL_NAME]
+        if not is_forum_channel(python_help_channel_id, discord_client):
+            delete_channel(python_help_channel_id, discord_client)
+        else:
+            create_help_channel = False
+
+    if create_help_channel:
+        python_help_channel_name = PYTHON_HELP_CHANNEL_NAME.replace('_', '-')
+        python_help_channel_id = create_forum_channel(python_help_channel_name, GUILD_ID, discord_client)
```

Comment:
After this line, `all_channels` should be updated to use the new channel ID, otherwise you get:
![image](https://user-images.githubusercontent.com/18114431/228802197-320293d3-0111-4486-a341-17e095031910.png)

Reply:
> Also I would personally put all of the client-dependent functions inside the client class, but it's more of an opinion.

Yeah it's better that way, it's just that the original design wasn't intended to have a client like this & drifted away (and my thoughts with it as well :P)
ae3db03f95eb157db2c1c11787e7c1725d7883e6

Reply:
01c2fc14631dff8f84a3695b788317ccb0d02030